packages/live-status-gateway/package.json

```diff
@@ -0,0 +1,79 @@
+{
+	"name": "live-status-gateway",
+	"version": "1.49.0-in-development",
+	"private": true,
+	"description": "Provides state from Sofie over sockets",
+	"license": "MIT",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/nrkno/tv-automation-server-core.git",
+		"directory": "packages/live-status-gateway"
+	},
+	"bugs": {
+		"url": "https://github.com/nrkno/tv-automation-server-core/issues"
+	},
+	"homepage": "https://github.com/nrkno/tv-automation-server-core/blob/master/packages/live-status-gateway#readme",
+	"contributors": [],
+	"scripts": {
+		"build": "rimraf dist && yarn build:main",
+		"buildstart": "yarn build && yarn dev",
+		"buildinspect": "yarn build && yarn inspect",
+		"build:main": "tsc -p tsconfig.build.json",
+		"lint:raw": "eslint --ext .ts --ext .js --ignore-pattern dist",
+		"lint": "yarn lint:raw .",
+		"unit": "jest",
+		"test": "yarn lint && yarn unit",
+		"watch": "jest --watch",
+		"cov": "jest; open-cli coverage/lcov-report/index.html",
+		"cov-open": "open-cli coverage/lcov-report/index.html",
+		"ci": "yarn test",
+		"gendocs": "rimraf docs && ag ./api/asyncapi.yaml @asyncapi/html-template -o docs -p singleFile=true -p outFilename=liveStatus.html",
+		"genserver": "rimraf server && ag ./api/asyncapi.yaml @asyncapi/nodejs-ws-template -o server -p server=development",
+		"inspect": "node --experimental-fetch --inspect dist/index.js",
+		"start": "node --experimental-fetch dist/index.js",
+		"validate:dependencies": "yarn audit --groups dependencies && yarn license-validate",
+		"validate:dev-dependencies": "yarn audit --groups devDependencies",
+		"license-validate": "yarn sofie-licensecheck"
+	},
+	"engines": {
+		"node": "^14.18 || ^16.14 || ^18.5"
+	},
+	"keywords": [
+		"broadcast",
+		"socket",
+		"typescript",
+		"javascript",
+		"open",
+		"source",
+		"automation",
+		"ncs",
+		"nrcs",
+		"rundown",
+		"production"
+	],
+	"dependencies": {
+		"@sofie-automation/blueprints-integration": "link:../blueprints-integration",
+		"@sofie-automation/server-core-integration": "link:../server-core-integration",
+		"@sofie-automation/corelib": "link:../corelib",
+		"@sofie-automation/shared-lib": "link:../shared-lib",
```

Comment:
It is my understanding that these sorts of links don't work very well. Apparently "*" should work, since these are all present in the workspace?

Reply:
yes, or whatever we do in the other gateways.

We have to do this in some places, as meteor isn't part of the workspace, but meteor wont be importing any code from this package